__tests__/app.test.js

```diff
@@ -35,3 +35,26 @@ describe("/api", () => {
         });
     });
 });
+describe("/api/articles", () => {
+    test("GET: 200 sends an array of article objects", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).toBeInstanceOf(Array);
+          articles.forEach((article) => {
+            expect(article).toMatchObject({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_img_url: expect.any(String),
+              comment_count: expect.any(String)
+            })
+        })
+    })
```

Comment:
You should be checking the articles are coming back in created_at order descending by default. Please add a test for this. You can use the `jest -sorted` library for this 👀